supplementary_style_guide/glossary_terms_conventions/general_conventions/d.adoc

```diff
@@ -613,7 +613,7 @@ Red Hat Directory Server conforms to LDAP standards.
 [discrete]
 [[drools-expert]]
 ==== image:images/yes.png[yes] Drools Expert (noun)
-*Description*: The "Drools Expert" is a pattern matching-based rule engine that runs on Java EE application servers, Red Hat JBoss BRMS platform, or bundled with Java applications. It comprises an inference engine, a production memory, and a working memory. Rules are stored in the production memory, and the facts that the inference engine matches the rules against are stored in the working memory.
+*Description*: The +Drools Expert+ is a pattern matching-based rule engine that runs on Java EE application servers, Red Hat JBoss BRMS platform, or bundled with Java applications. It comprises an inference engine, a production memory, and a working memory. Rules are stored in the production memory, and the facts that the inference engine matches the rules against are stored in the working memory.
```

Comment:
The plus signs here should be underscores

Reply:
:+1: